demo/kubernetes/rook-pvcontroller.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
```

Comment:
This will not be needed once you build the Rook PV provisioner into the operator.
No need to build something into k8s for removing this.